packages/collections/sort.pony

```diff
@@ -1,6 +1,21 @@
 primitive Sort[A: Seq[B] ref, B: Comparable[B] #read]
   """
   Implementation of dual-pivot quicksort.
```

Comment:
If you want to, explaining a bit about quicksort and what its appropriate for, wouldn't be out of line.

The issue was for the example but feel free to go beyond that in enhancing documentation if you want.

Reply:
👍 

I'm on it.  I'm guessing it's cool to call out to wikipedia for more info too...

(I added expected the output too.)